packages/perseus/src/widgets/__tests__/expression.test.ts

```diff
@@ -47,7 +45,6 @@ describe("Expression Widget", function () {
         jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
             testDependencies,
         );
-        registerAllWidgetsForTesting();
```

Comment:
We call this function in `renderQuestion` and all of the tests in this file use that helper, so this is redundant.